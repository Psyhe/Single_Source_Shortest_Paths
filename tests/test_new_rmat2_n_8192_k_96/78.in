8192 6662 6746
6662 1540 198
6662 4884 11
6662 2692 102
6662 7824 223
6662 2592 99
6662 6150 88
6662 7062 83
6662 6050 208
6662 7735 185
6662 7747 228
6662 6182 150
6662 3594 86
6662 2054 94
6662 4111 130
6662 2714 97
6662 6181 63
6662 5678 134
6662 6726 196
6662 6658 34
6663 6183 30
6663 6149 89
6663 3 69
6663 2565 149
6663 4775 158
6663 4865 11
6663 6659 35
6663 550 159
6663 7013 250
6663 6727 136
6663 3075 242
6663 2631 115
6664 6154 156
6664 2568 242
6664 6760 44
6664 6656 30
6664 2688 188
6664 6796 124
6664 6665 133
6664 2601 163
6664 7180 55
6664 4113 27
6664 6155 75
6664 7176 152
6664 4616 227
6664 7084 27
6664 2568 100
6664 6282 98
6664 7688 65
6664 6665 203
6664 6168 113
6664 7821 70
6664 6234 108
6665 8040 111
6665 6916 126
6665 4617 126
6665 7690 138
6665 6664 133
6665 2701 245
6665 6920 252
6665 1617 10
6665 2153 128
6665 2761 185
6665 7697 158
6665 8089 253
6665 6977 179
6665 6882 73
6665 3082 33
6665 6169 240
6665 6664 203
6665 7025 26
6665 4617 254
6665 6977 197
6665 4633 154
6665 7177 180
6665 7937 250
6666 5642 14
6666 6154 63
6666 6656 213
6666 6994 203
6666 708 140
6666 6794 164
6666 6750 186
6666 138 233
6666 12 74
6666 6686 189
6666 6163 97
6666 6154 147
6666 2570 116
6666 6170 252
6666 4750 146
6666 6810 195
6666 6794 44
6666 6920 220
6666 4098 241
6666 2842 40
6666 2569 235
6666 4378 234
6666 5646 33
6666 5320 6
6667 6683 3
6667 6719 83
6667 7343 168
6667 2969 218
6667 810 125
6667 7983 147
6667 4995 59
6667 2571 93
6667 7691 20
6667 1035 170
6667 6153 234
6667 2587 83
6667 2600 175
6667 3659 134
6668 7020 106
6668 8 21
6668 7812 33
6668 6956 79
6668 6152 28
6668 2954 20
6668 6702 163
6668 5668 161
6668 6701 183
6668 6680 255
6668 6175 192
6668 6949 58
6668 6732 39
6668 4620 150
6668 6734 218
6669 7721 59
6669 7689 140
6669 7173 13
6669 2575 250
6669 6949 95
6669 6661 90
6669 4717 236
6669 6167 159
6669 6989 10
6669 6700 127
6669 7752 94
6669 6724 83
6670 2184 168
6670 6756 118
6670 6223 119
6670 2586 23
6670 2564 219
6670 6766 75
6670 3366 126
6670 2822 108
6670 6790 82
6670 8086 2
6670 7814 203
6670 2566 245
6670 7212 248
6670 526 182
6670 2570 243
6670 6158 90
6670 2670 50
6671 6841 24
6671 6735 111
6671 6221 236
6671 2575 222
6671 7695 197
6671 4650 93
6671 2063 145
6671 2054 40
6671 4138 209
6671 5915 133
6672 6769 152
6672 530 74
6672 3608 227
6672 4630 117
6672 6736 96
6672 6680 251
6672 4609 127
6672 4112 20
6672 6741 18
6672 6706 25
6672 7188 79
6672 6801 92
6672 6802 19
6672 4864 108
6672 3601 182
6672 2074 132
6672 6704 152
6672 4272 58
6672 7688 86
6672 6146 192
6672 6792 113
6672 6707 174
6672 5906 120
6672 6684 121
6672 4240 131
6672 6738 138
6672 2064 146
6672 2576 211
6672 4656 246
6673 2832 159
6673 6681 69
6673 2689 93
6673 4629 83
6673 6802 40
6673 6657 188
6673 2579 186
6673 6705 238
6673 5233 194
6673 6658 164
6673 6677 93
6673 5769 29
6673 2707 247
6673 6144 79
6673 6684 184
6673 785 203
6673 2577 183
6673 6177 64
6673 6808 108
6673 1557 178
6673 6417 103
6673 6739 248
6673 3601 231
6673 5632 211
6673 4627 131
6673 6675 170
6673 6961 238
6673 6657 169
6674 6918 136
6674 6296 234
6674 6688 175
6674 6258 231
6674 6786 84
6674 6710 232
6674 6658 173
6674 1538 142
6674 6419 133
6674 6790 235
6674 7706 176
6674 5146 7
6674 5170 93
6674 7318 13
6674 2611 255
6674 6160 44
6674 5142 54
6674 6802 198
6674 3614 249
6674 2578 60
6674 6738 153
6674 6162 237
6674 6768 125
6674 6146 176
6674 4630 7
6674 6162 206
6674 6168 125
6674 4690 181
6675 7425 255
6675 2067 232
6675 2723 158
6675 4625 137
6675 6189 80
6675 6163 185
6675 6673 170
6675 4626 145
6675 2067 147
6675 2067 140
6675 7698 15
6675 4251 116
6675 6811 18
6676 3618 248
6676 6164 174
6676 6932 48
6676 6964 101
6676 2724 55
6676 7685 127
6676 6684 36
6676 4692 190
6676 4688 245
6676 3868 230
6676 4692 175
6676 6962 209
6676 4176 18
6676 6742 0
6676 5076 174
6676 4658 21
6676 5653 39
6676 6804 139
6676 8022 95
6676 7852 201
6677 4117 76
6677 3584 207
6677 4625 31
6677 5920 252
6677 6917 64
6677 6740 244
6677 1813 163
6677 6149 186
6677 6997 30
6677 7701 35
6677 1029 205
6677 5044 243
6677 4629 152
6677 2135 91
6677 6679 241
6677 6673 93
6678 6684 34
6678 3602 121
6678 4756 98
6678 5638 62
6678 2262 95
6678 2582 82
6679 2135 62
6679 4182 136
6679 6677 241
6679 4614 114
6679 6146 77
6679 6167 53
6679 2102 246
6680 6924 37
6680 2625 14
6680 4617 251
6680 6672 251
6680 4904 121
6680 6668 255
6680 6425 41
6680 6152 33
6680 6968 62
6680 6169 111
6680 6296 8
6680 4632 139
6680 5902 245
6680 6736 252
6680 541 201
6680 6730 136
6681 6685 241
6681 265 184
6681 2577 7
6681 6673 69
6681 7004 100
6681 4689 102
6681 2619 170
6681 7229 191
6681 6707 48
6681 4633 204
6681 5646 29
6681 6289 167
6681 2685 14
6681 2649 130
6681 7691 35
6681 6801 232
6682 6705 97
6682 4632 22
6682 6718 195
6682 5016 45
6682 571 78
6682 5650 196
6682 2840 139
6682 4656 251
6682 4627 147
6682 2072 152
6683 6163 172
6683 6723 41
6683 6667 3
6683 5145 209
6683 4414 138
6683 6712 149
6683 7770 2
6683 3003 146
6683 6731 39
6684 7940 61
6684 1556 149
6684 7692 106
6684 4612 125
6684 5655 192
6684 8004 3
6684 6332 36
6684 6672 121
6684 4670 229
6684 6168 129
6684 6673 184
6684 6196 234
6684 4752 26
6684 2712 182
6684 6780 115
6684 6676 36
6684 6678 34
6684 6904 164
6684 7004 200
6684 6942 209
6684 6685 133
6684 6708 124
6685 4621 74
6685 6661 124
6685 2757 62
6685 2108 55
6685 7632 82
6685 6681 241
6685 6684 133
6685 7245 91
6685 3676 230
6685 2824 203
6686 2575 238
6686 2603 164
6686 4620 241
6686 4878 162
6686 7702 221
6686 7740 168
6686 6666 189
6687 4381 17
6687 7700 212
6687 31 24
6687 2333 48
6688 6660 50
6688 7712 33
6688 6788 142
6688 3969 77
6688 3616 165
6688 7745 182
6688 6176 75
6688 7748 234
6688 6880 83
6688 5892 113
6688 2608 125
6688 6754 13
6688 4320 207
6688 3632 205
6688 6656 83
6688 6176 68
6688 1704 242
6688 6674 175
6688 4624 88
6688 2912 104
6688 5152 120
6688 6690 50
6688 2560 159
6688 6945 236
6688 7940 146
6688 4768 33
6688 6944 90
6688 33 146
6688 6177 119
6688 1604 249
6688 7722 106
6688 7808 232
6688 6944 140
6688 6324 245
6688 6402 79
6688 2064 214
6688 6241 3
6688 6946 146
6688 6192 52
6688 4842 100
6688 4896 224
6688 2688 188
6688 2096 64
6688 7946 86
6688 6304 172
6688 6657 20
6688 6691 200
6689 2729 90
6689 2560 196
6689 6691 71
6689 6657 249
6689 7715 246
6689 4641 50
6689 5665 107
6689 4657 161
6689 7745 126
6689 6695 76
6689 2595 59
6689 4641 101
6689 2561 91
6689 6241 131
6689 4610 57
6689 6573 151
6689 6435 104
6689 673 222
6689 7208 190
6689 7936 169
6689 6161 202
6689 4129 176
6689 2321 226
6689 6816 236
6689 6273 119
6690 7715 84
6690 4658 183
6690 4644 184
6690 4098 167
6690 6950 176
6690 6688 50
6690 4128 36
6690 546 230
6690 4642 245
6690 7215 113
6690 2562 219
6690 4650 186
6690 2848 212
6690 6706 184
6690 7086 67
6690 5650 21
6690 2856 197
6690 6658 91
6690 2594 118
6690 6818 176
6690 7426 20
6690 4642 234
6690 2722 0
6690 4450 89
6690 6178 70
6691 4963 94
6691 4256 117
6691 4739 119
6691 6201 107
6691 6177 203
6691 6819 105
6691 6689 71
6691 4651 184
6691 6695 147
6691 4659 78
6691 6715 148
6691 7014 28
6691 4903 112
6691 33 169
6691 6945 22
6691 6688 200
6691 167 222
6691 4867 249
6692 6716 193
6692 3588 238
6692 6700 156
6692 6192 103
6692 6799 96
6692 6432 119
6692 6820 152
6692 7716 134
6692 5677 160
6692 6188 6
6692 6252 28
6692 6660 234
6692 2081 72
6692 4646 24
6693 5737 166
6693 556 184
6693 7429 52
6693 6945 126
6693 7842 49
6693 2864 22
6693 4645 152
6693 7735 120
6693 7783 80
6693 807 26
6693 6177 35
6693 2213 196
6693 4645 49
6693 4643 237
6693 2085 8
6693 5 115
6693 6437 36
6693 2605 66
6693 1061 28
6693 2085 238
6694 6822 188
6694 6660 33
6694 6804 219
6694 4130 104
6694 4144 81
6694 2564 126
6694 7734 145
6694 6959 185
6694 4711 43
6694 102 64
6694 7726 6
6694 2343 215
6694 2102 137
6695 6704 94
6695 6708 114
6695 6197 252
6695 6769 109
6695 4711 139
6695 4231 190
6695 2571 14
6695 3639 164
6695 6691 147
6695 8119 104
6695 6689 76
6696 4648 46
6696 872 57
6696 4648 6
6696 6152 215
6696 3320 184
6696 7016 145
6696 4938 86
6696 6937 14
6696 6712 88
6696 4640 225
6696 552 120
6696 548 26
6696 4792 221
6696 4652 30
6696 2600 5
6696 6697 237
6696 2568 18
6696 6827 204
6696 2882 207
6696 6184 110
6696 7008 194
6696 3628 50
6697 4777 39
6697 6707 146
6697 2593 154
6697 4353 196
6697 7273 141
6697 4717 211
6697 6861 202
6697 6158 8
6697 2568 40
6697 6184 254
6697 6696 237
6697 6701 199
6697 553 30
6697 265 220
6698 4650 84
6698 4395 56
6698 4666 198
6698 5178 198
6698 2666 60
6698 7066 240
6698 2354 68
6698 38 243
6698 7710 21
6698 6186 48
6698 3594 193
6698 4131 173
6698 7210 42
6698 802 164
6698 4720 253
6698 7202 92
6699 553 96
6699 6791 118
6699 6243 180
6699 6274 216
6699 555 250
6699 6643 131
6699 7739 232
6699 4843 33
6700 6717 70
6700 6669 127
6700 6180 172
6700 2611 181
6700 6692 156
6700 2063 72
6700 7692 73
6700 556 210
6700 2604 29
6700 2088 231
6700 6708 132
6700 6660 208
6700 7292 8
6700 300 201
6700 7950 78
6701 6668 183
6701 2605 124
6701 6697 199
6701 6709 193
6701 6927 236
6701 2173 74
6701 2153 2
6701 2860 61
6701 2873 250
6702 7079 247
6702 7742 2
6702 174 239
6702 6750 80
6702 6668 163
6702 6942 113
6702 1829 133
6702 2127 145
6702 7086 147
6702 2606 88
6703 3626 4
6703 6190 160
6703 4643 231
6703 4641 9
6703 519 127
6703 6959 243
6703 2589 35
6704 6708 101
6704 7728 9
6704 6672 152
6704 6695 94
6704 688 212
6704 4212 62
6704 2096 192
6704 1840 159
6704 7024 6
6704 6832 123
6704 6706 94
6704 2593 126
6704 1200 135
6705 4657 33
6705 6193 51
6705 6993 252
6705 6448 15
6705 5665 96
6705 7984 252
6705 4145 112
6705 2737 0
6705 7719 204
6705 6673 238
6705 6682 97
6705 3620 153
6705 6656 227
6706 2592 120
6706 6672 25
6706 4901 199
6706 7866 92
6706 6198 233
6706 6690 184
6706 6704 94
6706 6974 179
6706 2662 82
6706 5682 176
6706 3159 215
6706 2720 225
6706 2612 46
6706 2858 13
6706 5922 4
6706 7714 147
6707 2097 222
6707 4882 37
6707 3729 60
6707 6842 230
6707 6681 48
6707 3267 254
6707 6672 174
6707 2675 244
6707 8111 175
6707 6995 82
6707 6937 210
6707 6697 146
6707 6963 172
6708 6709 222
6708 4372 89
6708 4133 228
6708 2228 91
6708 4918 56
6708 6260 131
6708 6196 129
6708 7478 219
6708 6704 101
6708 6684 124
6708 6700 132
6708 6820 127
6708 6695 114
6708 4889 95
6708 4756 42
6708 2566 238
6708 5680 170
6708 2620 60
6709 119 236
6709 6196 47
6709 2743 24
6709 4245 176
6709 7869 200
6709 6701 193
6709 6197 160
6709 6836 179
6709 2621 113
6709 6839 145
6709 6708 222
6709 2613 92
6710 6674 232
6710 6773 57
6710 3943 32
6710 7270 173
6710 6838 169
6710 6190 89
6710 7092 6
6710 6658 184
6710 2614 91
6710 6838 173
6710 7734 169
6710 4854 243
6711 145 148
6711 6196 178
6711 4667 161
6711 7731 17
6711 6963 208
6711 6257 117
6712 2080 183
6712 4624 193
6712 4657 203
6712 6962 213
6712 6204 93
6712 6683 149
6712 2233 135
6712 4728 242
6712 6192 157
6712 2621 242
6712 6960 56
6712 6458 208
6712 2864 91
6712 7724 146
6712 2608 79
6712 7736 2
6712 7856 15
6712 6803 236
6712 6330 183
6712 6696 88
6712 3644 18
6712 2858 232
6713 572 233
6713 6200 18
6713 569 111
6713 6204 49
6714 2843 88
6714 6746 146
6714 5704 103
6714 6170 201
6714 4614 145
6714 6328 159
6714 6202 83
6715 6203 159
6715 6463 199
6715 6691 148
6715 4875 172
6715 7551 168
6716 6204 48
6716 6452 233
6716 570 72
6716 6692 193
6716 6718 146
6716 4632 23
6716 4860 181
6716 5688 99
6716 6445 245
6716 2748 123
6716 4732 165
6717 7285 54
6717 2591 103
6717 4729 125
6717 6700 70
6717 2624 18
6717 2609 182
6717 7189 183
6718 7966 120
6718 6682 195
6718 2622 195
6718 7998 53
6718 5791 229
6718 6188 83
6718 6719 4
6718 2842 226
6718 6716 146
6719 2623 66
6719 4975 136
6719 4799 227
6719 6718 4
6719 6667 83
6720 6148 43
6720 128 36
6720 6736 131
6720 6273 253
6720 4672 63
6720 6656 32
6720 2632 95
6720 6400 158
6720 6737 123
6720 5124 119
6720 4676 205
6720 2624 254
6720 7684 232
6720 5696 25
6720 7712 166
6720 8128 176
6720 6208 203
6720 4680 80
6720 2632 253
6720 2120 25
6720 2048 210
6720 2880 22
6720 7825 140
6720 6403 216
6720 6208 163
6721 6753 239
6721 8003 85
6721 5187 67
6721 4929 74
6721 7681 113
6721 6729 77
6721 2881 69
6721 6724 126
6721 4162 153
6721 6745 118
6721 6339 211
6721 7745 139
6721 6897 52
6721 2828 14
6721 2561 111
6721 6209 58
6721 2624 184
6722 6724 15
6722 5698 83
6722 2758 163
6722 7746 147
6722 8034 171
6722 6223 255
6722 2880 142
6722 6755 119
6722 3922 226
6722 4610 56
6722 7744 62
6722 6243 22
6722 6882 114
6722 6336 89
6722 4114 72
6722 4211 62
6722 6754 153
6722 4672 24
6722 7238 212
6722 6851 46
6722 6102 151
6722 6659 130
6722 2634 185
6722 7682 33
6722 6210 89
6722 2682 15
6722 4674 155
6723 6219 128
6723 2637 184
6723 6443 23
6723 5185 91
6723 4675 230
6723 6658 199
6723 2626 46
6723 6683 41
6723 1603 111
6723 259 216
6723 7749 188
6723 6771 134
6723 2752 192
6723 6731 248
6723 5026 197
6723 6753 164
6723 6219 87
6724 7168 192
6724 5188 31
6724 7173 114
6724 5700 162
6724 5205 159
6724 4164 181
6724 5732 199
6724 6976 236
6724 6721 126
6724 2148 88
6724 6669 83
6724 2563 245
6724 4100 56
6724 7758 160
6724 4672 81
6724 7748 74
6724 6732 139
6724 6220 3
6724 6212 93
6724 6740 106
6724 6722 15
6724 4164 226
6724 5956 212
6724 6740 51
6724 2897 87
6725 3657 113
6725 6765 12
6725 2917 94
6725 3397 237
6725 2625 117
6725 2624 41
6725 609 116
6725 2121 0
6725 6181 25
6725 2657 86
6726 6978 157
6726 6860 87
6726 6978 36
6726 3654 145
6726 4220 178
6726 6662 196
6726 6742 76
6726 6246 34
6726 4613 240
6726 4675 4
6726 6918 129
6726 4678 19
6726 6215 155
6726 1635 9
6726 7403 243
6726 2733 28
6726 6742 145
6727 6925 51
6727 631 53
6727 6663 136
6727 2653 232
6727 6210 191
6727 5159 223
6727 7874 119
6727 6774 16
6727 871 106
6727 6215 82
6728 6344 222
6728 96 106
6728 6792 14
6728 2632 109
6728 4680 215
6728 7114 119
6728 6156 254
6728 4616 121
6728 4953 176
6728 2632 110
6728 5192 176
6728 7874 28
6728 6500 55
6728 5652 66
6728 2120 236
6728 2124 3
6728 7880 219
6728 7880 170
6728 4616 43
6728 6736 119
6728 2122 23
6728 4680 187
6728 7816 174
6728 6984 108
6728 6216 121
6728 2568 194
6728 6234 45
6729 2697 55
6729 2897 141
6729 5641 125
6729 2841 136
6729 3649 207
6729 6353 115
6729 6755 164
6729 6745 211
6729 4680 5
6729 6721 77
6729 4169 158
6729 5777 153
6729 6733 98
6730 2638 20
6730 6146 52
6730 6158 134
6730 5065 179
6730 6216 35
6730 7822 81
6730 4106 168
6730 6474 249
6730 3784 163
6730 7374 56
6730 2632 161
6730 7944 165
6730 6680 136
6730 6858 87
6730 6810 87
6731 6219 119
6731 6763 219
6731 2939 240
6731 5033 79
6731 6683 39
6731 2883 113
6731 5709 33
6731 6723 248
6732 2638 3
6732 7748 190
6732 6980 99
6732 2122 74
6732 6668 39
6732 3604 16
6732 2628 58
6732 6724 139
6732 6656 170
6732 4684 27
6732 6764 159
6732 7756 197
6732 6894 208
6733 6657 51
6733 4679 180
6733 6269 108
6733 4171 167
6733 7693 166
6733 2637 184
6733 4703 98
6733 6729 98
6733 6208 191
6733 6761 30
6733 7501 232
6733 4892 201
6734 6798 182
6734 6150 143
6734 6660 228
6734 6668 218
6734 2894 156
6734 2764 252
6734 4843 63
6734 2378 202
6734 1032 53
6734 2575 238
6735 6987 209
6735 2151 45
6735 6989 95
6735 6213 19
6735 6671 111
6735 69 99
6735 5197 39
6735 7119 198
6735 7759 95
6736 606 245
6736 4688 169
6736 2754 255
6736 6168 235
6736 7232 220
6736 6992 223
6736 6680 252
6736 596 22
6736 6720 131
6736 7120 71
6736 8016 189
6736 6848 101
6736 6488 34
6736 7700 112
6736 6400 245
6736 6184 187
6736 6744 63
6736 2978 38
6736 4688 82
6736 6996 195
6736 2576 136
6736 2644 209
6736 5720 180
6736 1236 157
6736 4624 127
6736 2624 235
6736 6248 145
6736 4676 245
6736 402 249
6736 6672 96
6736 6656 179
6736 6164 75
6736 6728 119
6736 6464 124
6736 4688 192
6736 6168 136
6736 2673 171
6737 3701 59
6737 513 200
6737 85 253
6737 2677 191
6737 6720 123
6737 4858 171
6737 6740 158
6737 6225 202
6737 3668 54
6737 6740 191
6737 2241 79
6737 6227 102
6737 4696 92
6737 208 121
6737 6229 29
6738 2626 137
6738 6672 138
6738 4689 167
6738 2579 89
6738 6674 153
6738 6994 65
6738 4695 47
6738 4626 162
6738 26 255
6738 6856 255
6738 7003 113
6738 2586 173
6738 6842 54
6738 7682 207
6738 66 63
6738 6976 219
6738 6994 61
6738 2642 238
6738 82 175
6738 18 113
6738 6214 186
6739 2586 6
6739 4339 121
6739 4691 10
6739 6867 181
6739 2803 171
6739 1623 203
6739 1600 183
6739 7762 134
6739 8019 74
6739 7763 177
6739 4691 97
6739 6673 248
6740 6992 229
6740 7362 120
6740 84 117
6740 6737 158
6740 6737 191
6740 5204 123
6740 4886 89
6740 4810 82
6740 2644 174
6740 6228 84
6740 7008 229
6740 2646 59
6740 2758 195
6740 6166 175
6740 612 94
6740 6724 106
6740 6677 244
6740 6724 51
6740 4164 199
6740 7765 70
6741 2260 246
6741 721 154
6741 4180 197
6741 3905 144
6741 2702 25
6741 6109 55
6741 6672 18
6741 7124 27
6742 6726 145
6742 7376 28
6742 4694 12
6742 2654 128
6742 7702 47
6742 6783 218
6742 7773 32
6742 6726 76
6742 6751 143
6742 2582 168
6742 6743 31
6742 7762 135
6742 6676 0
6743 6742 31
6743 6855 116
6743 2630 236
6744 3416 153
6744 7680 155
6744 2257 187
6744 8024 80
6744 7512 7
6744 6940 37
6744 4190 79
6744 600 113
6744 2696 177
6744 7004 117
6744 4824 123
6744 6736 63
6744 2764 243
6744 2584 157
6744 6936 38
6744 6156 187
6745 5753 237
6745 6721 118
6745 6225 83
6745 4701 203
6745 601 39
6745 713 239
6745 5585 20
6745 6729 211
6746 6392 22
6746 2648 77
6746 6218 77
6746 4618 253
6746 6714 146
6746 1626 241
6746 6466 59
6746 7762 207
6746 7000 246
6746 6216 205
6746 6858 132
6746 2890 225
6746 2890 6
