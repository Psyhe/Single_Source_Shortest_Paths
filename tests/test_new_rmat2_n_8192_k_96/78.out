91
29
63
73
42
106
52
51
82
62
44
83
51
112
42
77
108
113
39
58
45
109
74
82
217
55
50
70
73
101
64
62
80
72
26
54
82
76
86
29
71
29
52
20
57
104
109
102
140
114
45
82
147
216
46
55
141
145
50
41
48
86
35
51
69
77
44
37
47
148
30
82
81
53
70
87
71
71
86
62
42
73
75
82
86
